src/components/Header/index.js

```diff
@@ -82,39 +116,45 @@ const Header = () => {
             </div>
             <div className="col-3 col-lg-3">
               <div className="header-col justify-content-end buttons">
-                {/* >= lg */}
-                <a
-                  href="https://community.debtcollective.org/signup"
-                  target="_blank"
-                  rel="noopener noreferrer"
-                  role="button"
-                  onClick={trackOutboundLink}
-                  className="btn btn-lg btn-outline-dark d-none d-xl-block btn-session"
-                >
-                  Sign up
-                </a>
-                {/* >= md */}
-                <a
-                  href="https://community.debtcollective.org/login"
-                  target="_blank"
-                  rel="noopener noreferrer"
-                  role="button"
-                  onClick={trackOutboundLink}
-                  className="btn btn-primary btn-lg d-none d-md-block btn-session "
-                >
-                  Login
-                </a>
-                {/* small */}
-                <a
-                  href="https://community.debtcollective.org/login"
-                  target="_blank"
-                  rel="noopener noreferrer"
-                  role="button"
-                  onClick={trackOutboundLink}
-                  className="btn btn-primary btn-sm d-md-none d-xs-block d-sm-block btn-session"
-                >
-                  Login
-                </a>
+                {isLoggedIn ? (
+                  <Profile user={user} />
+                ) : (
+                  <>
+                    {/* >= lg */}
+                    <a
+                      href={`${process.env.GATSBY_COMMUNITY_URL}/session/sso_cookies/signup?${redirectParam}`}
```

Comment:
@duranmla I suggest two variables here instead of hard-coding the URL, one for login and one for signup. These URLs are not stable yet and may change.